src/test/java/org/fundacionjala/coding/manuel/CesarVigenereCryptoTest.java

```diff
@@ -0,0 +1,140 @@
+package org.fundacionjala.coding.manuel;
+
+import org.fundacionjala.coding.manuel.encryptionalgorithms.Cesar;
+import org.fundacionjala.coding.manuel.encryptionalgorithms.Crypto;
+import org.fundacionjala.coding.manuel.encryptionalgorithms.Vigenere;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Class to test CesarVigenereCrypto.
+ */
+public class CesarVigenereCryptoTest {
+
+    /**
+     * Object of the class Cesar.
+     */
+    private Crypto cesarCrypto;
+
+    /**
+     * Object of the class Vigenere.
+     */
+    private Crypto vigenereCrypto;
+
+    private static final String VIGENERE_KEY = "LIMON";
+    private static final String TEST_WORD = "ZORRO";
+
+    /**
+     * Method to init object.
+     */
+    @Before
+    public void init() {
+        cesarCrypto = new Cesar();
+        vigenereCrypto = new Vigenere();
```

Comment:
This is wrong
because you are not using the two instances in all the tests

Consider splitting in two unit test class